picard/ui/theme.py

```diff
@@ -204,37 +199,49 @@ def update_palette(self, palette, dark_theme, accent_color):
     theme = WindowsTheme()
 
 elif IS_MACOS:
-    try:
-        import AppKit
-    except ImportError:
-        AppKit = None
+    dark_appearance = False
+    if OS_SUPPORTS_THEMES:
```

Comment:
I think it would be better to test for `AppKit` even though at the moment `OS_SUPPORTS_THEMES` is always `False` when `AppKit` is unavailable.
It would be much cleaner and future-proof to enclose code depending on `AppKit` in a `if AppKit:`

To me, `OS_SUPPORTS_THEMES` is another thing.

Reply:
One reason to check for OS_SUPPORTS_THEMES is that the following code is not needed if that is false, independent of whether AppKit would be available or not. The fully correct check would probably be `if OS_SUPPORTS_THEMES  and AppKit`, but as OS_SUPPORTS_THEMES  is always false if no AppKit is present I skipped that.

Reply:
> OS_SUPPORTS_THEMES is always false if no AppKit is present

Actually that was my point, because if the expression defining `OS_SUPPORTS_THEMES` changes, it might not be true in the future, so I was thinking it would be safer to have explicit test for Appkit.

Reply:
I have changed the checks to `OS_SUPPORTS_THEMES and AppKit` for code that makes direct use of AppKit. You are right, that could prevent bugs on future refactorings. `OS_SUPPORTS_THEMES ` is not necessarily tied to AppKit, that's just how it turns out to work currently.